docs/compiling-mono/linux/index.md

```diff
@@ -15,6 +15,22 @@ Building Mono is a very simple process as Mono has very few external dependencie
 
 On some distribution you will need to install some development tools which are not installed by default.
 
+Compiler/Build dependencies
+---------------------
+
+Mono depends on GCC 4.8+ or Clang 3.3+ (it needs to support C++11) and on CMake 2.8.10.
```

Comment:
We don't need cmake if you disable BTLS (at the moment, I guess this will change once your System.Native work lands right?)